pkg/app/web/src/components/settings-page/piped/components/piped-table-row.tsx

```diff
@@ -150,7 +150,9 @@ export const PipedTableRow: FC<Props> = memo(function PipedTableRow({
             <CopyIconButton name="Piped ID" value={piped.id} />
           </Box>
         </TableCell>
-        <TableCell>{piped.version}</TableCell>
+        <TableCell>
+          {piped.version}{piped.desiredVersion.length > 0 && piped.desiredVersion !== piped.version ? ` (upgrading to ${piped.desiredVersion})` : ''}
```

Comment:
```suggestion
          {piped.version}{piped.desiredVersion && piped.desiredVersion !== piped.version && ` (upgrading to ${piped.desiredVersion})`}
```
nits

Reply:
Haha. Hello JS. I think we should keep using `? : ` for more understanding.
But agree about changing `piped.desiredVersion.length > 0 &&` to `piped.desiredVersion &&`

Reply:
Sure 👍  `.length` in case of null value cause `cannot read properties of null` so changing to truthy value pattern has its point. Lets change it as your suggestion.

Reply:
Applied!